source/backend/api/Repositories/Cdogs/CdogsRepository.cs

```diff
@@ -12,9 +12,10 @@
 using Microsoft.Extensions.Logging;
 using Pims.Api.Models.Cdogs;
 using Pims.Api.Models.CodeTypes;
-
 using Pims.Api.Models.Requests.Http;
 
+using FileTypes = Pims.Api.Models.Cdogs.FileTypes;
```

Comment:
is this necessary? seems like it may be unused.

Reply:
Pims.Api.Models.CodeTypes.FileTypes and Pims.Api.Models.Cdogs.FileTypes are ambiguous so this is needed.